_posts/2016-07-28-community-testing-day-on-20160805.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Community Testing Day on August 5th
+layout: post
+author: Chongrui Zhang
+---
+
+The Pulp 2.10 GA is [scheduled for Aug 16, 2016](https://pulp.plan.io/projects/pulp/wiki/Release_Schedule). To deliver a robust Pulp release, the Pulp QE and Pulp team have organized a Community Test day where the Pulp community can come together to test Pulp. This post further details the event and how to participate it.
+
+# What is a Community Testing Day and why host it?
+
+According to the Fedora blog, organizing a test day "is a great way to help expose your project and bring more testers to trialing a new update before it goes live".
+
+The main objective of hosting this Pulp Test Day is to identify as many issues as possible early on with the help of the community, get them fixed as quickly as possible and thus have a more stable release.
```

Comment:
s/issues/issues with 2.10/
Suggest ending the first sentence at `community` and replacing the second part with `Once identified, important issues can be fixed before the GA releases leading to a more stable release.`.